pkg/client/cmd/app.go

```diff
@@ -280,6 +280,13 @@ func appInfo(cmd *cobra.Command, args []string) {
 	color.New(color.FgCyan, color.Bold).Printf("[%s]\n", name)
 	bold := color.New(color.Bold).SprintFunc()
 
+	f, err := client.ReadConfigFile(cfgFile)
+	if err != nil {
+		client.PrintErrorAndExit("Cannot read the config file, have you created it with `teresa set-cluster` command?")
+	}
+
+	color.New(color.FgRed, color.Bold).Printf("env: %s\n", f.CurrentCluster)
+
 	fmt.Println(bold("team:"), info.Team)
```

Comment:
I'm in favor of  showing the environment but this way you are reading the config file two times (the first one in `connection.New`).

Reply:
Maybe saving the `current` cluster in some variable to use afterwards.

Either case, this information would be a cluster nickname and not an environment necessarily.